src/DynamoCore/Logging/DynamoLogger.cs

```diff
@@ -209,19 +209,13 @@ private void Log(string message, LogLevel level, bool reportModification)
         {
             lock (this.guardMutex)
             {
-                // In test mode, write the logs only to std out. 
-                if (testMode && !cliMode)
+                // In test mode or service mode, write the logs only to std out. 
+                if ((testMode && !cliMode) || serviceMode)
```

Comment:
so - since this still only allows std out logging will there be no issues with lambdas?

Reply:
that is correct